app/controllers/trips_controller.rb

```diff
@@ -16,9 +16,18 @@ def create
   end
 
   def edit
+    @trip = Trip.find(params[:id])
   end
 
   def update
+    @trip = Trip.find(params[:id])
+    @profile.user = current_user
```

Comment:
why are you setting the user of the profile here?

Reply:
yeah here you're right, we don't need it I think